tests/2018-06-01/Microsoft.Compute.tests.json

```diff
@@ -0,0 +1,1407 @@
+{
+  "tests": [
```

Comment:
>tests [](start = 3, length = 5)

we have another set of tests in /tools/templateFiles, which are basically some test templates. Can we add the test resources to TestTemplate.json and TestTemplate.preview.json? 

Should be just a copy/paste task.

Reply:
Sorry, can you explain this comment a bit more? Do you want me to copy everything in this test file to TestTemplate.json and TestTemplate.preview.json?